api/src/test/java/org/openmrs/module/pihcore/event/PatientEventConsumerTest.java

```diff
@@ -0,0 +1,56 @@
+package org.openmrs.module.pihcore.event;
+
+import org.apache.commons.dbutils.handlers.MapListHandler;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openmrs.module.dbevent.Database;
+import org.openmrs.module.dbevent.DbEventSource;
+import org.openmrs.module.dbevent.DbEventSourceConfig;
+import org.openmrs.module.dbevent.test.Mysql;
+import org.openmrs.module.dbevent.test.MysqlExtension;
+import org.openmrs.module.dbevent.test.TestEventContext;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(MysqlExtension.class)
+public class PatientEventConsumerTest {
+
+    @Test
```

Comment:
Note, @mogoodrich I haven't gotten unit tests working for this yet, and we'll need to discuss the approach given our CI situation (eg. how to use Docker effectively both as an execution environment for the Maven build and also use testcontainers in our unit tests).